src/Debug/axesViewer.ts

```diff
@@ -52,7 +52,7 @@ export class AxesViewer {
      * @param yAxis defines the node hierarchy used to render the y-axis
      * @param zAxis defines the node hierarchy used to render the z-axis
      */

```

Comment:
```suggestion
     * @param lineThickness The line thickness to use when creating the arrow. defaults to 1.
     */
```

Reply:
@mrlooi - You can merge this, unless you want to write your own documentation line :-)